src/components/Searchbar/Search.styles.js

```diff
@@ -62,9 +62,10 @@ const Wrapper = withTheme(styled.div`
     width: 100%;
     left: 0;
     background: ${themeGet('colors.base.white')};
-    box-shadow: ${themeGet('shadows.medium')};
+    box-shadow: 0px 60px 48px rgba(0, 0, 0, 0.09);
```

Comment:
Should we update or add this shadow to the theme config?

Reply:
@jordanwade I'm not too sure since it's the same as the medium shadow, just vertically positioned lower in respects to the center of the component and the shadow. I can add it in the config for now